src/components/attributeTable/AttributeTable.vue

```diff
@@ -15,19 +15,52 @@
         'items-per-page-options': [],
         'show-first-last-page': true
       }"
+    @click:row="onRowClick"
+    single-select
+    :value="selectedRow"
+    :item-key="uniqueRecordKeyName"
+    :items-per-page="rowsPerPage"
+    :height="getTableHeight()"
   ></v-data-table>
 </template>
 
 <script>
 import { Mapable } from '../../mixins/Mapable';
 import LayerUtil from '../../util/Layer';
+import { WguEventBus } from '../../WguEventBus';
+import SelectInteraction from 'ol/interaction/Select';
 
 export default {
   name: 'wgu-attributetable',
   mixins: [Mapable],
   props: {
+    /** The ID of the vector layer to display */
     layerId: {type: String, required: false, default: null},
-    loadingText: {type: String, required: false, default: 'Loading... Please wait'},
+
+    /** The loading text while the table is waiting for records */
+    loadingText: {type: String, required: false, default: 'Loading... Please move to an area where the layer is visible'},
+
+    /** The name of the unique feature identifier */
+    uniqueRecordKeyName: {type: String, required: false, default: 'fid'},
+
+    /**
+     * How many rows a page of the table should have.
+     * Should be manually adjusted with tableHeight.
+     */
+    rowsPerPage: {type: Number, required: false, default: 10},
+
+    /**
+     * The hight of the table in pixel.
+     * Should be manually adjusted with rowsPerPage.
+     */
+    tableHeight: {type: Number, required: false, default: 272},
+
+    /** If map and table should be synced */
+    activateTableMapInteraction: {type: Boolean, required: false, default: true},
```

Comment:
See comment above regarding naming of the property

Reply:
done with 73d37cc